web/client/translations/data.de-DE

```diff
@@ -440,216 +440,216 @@
+          "description": "Beschreibung:",
+          "groupName": "Gruppenname",
+          "groupDescription": "Beschreibung",
+          "saveGroup": "Speichern",
+          "createGroup": "Erstellen",
+          "creatingGroup": "Erstellen...",
+          "groupMembers": "Mitglied:",
+          "addMember": "Mitglied hinzufügen",
+          "noUsers": "Keine Benutzer für diese Gruppe gefunden",
+          "errorSaving": "Es gab einen Fehler beim Speichern dieser Gruppe",
+          "errorDelete": "Es gab einen Fehler beim Löschen dieser Gruppe",
+          "confirmDeleteGroup": "Bist du sicher, das du diese Gruppe löschen möchtest?"
         },
         "share":{
-            "title": "Share",
-            "titlePanel": "Share the map",
-            "socialIntro": "In your favourite social network",
-            "directLinkTitle": "Via a direct link",
-            "embeddedLinkTitle": "Via the embedded code",
-            "QRCodeLinkTitle": "qr code",
-            "msgCopiedUrl":"Copied",
-            "msgToCopyUrl":"Click to copy"
+            "title": "Teilen",
+            "titlePanel": "Teile die Karte",
+            "socialIntro": "In deinem bevorzugten sozialen Netzwerk",
+            "directLinkTitle": "Über einen direkten Link",
+            "embeddedLinkTitle": "Über einen eingebetteten Code",
+            "QRCodeLinkTitle": "QR code",
+            "msgCopiedUrl":"Kopiert",
+            "msgToCopyUrl":"Klicke um zu kopieren"
         },
         "snapshot": {
-            "title": "Snapshot Preview",
-            "save": "Save",
-            "tooltip": "Save a snapshot of the map.",
-            "googleBingError": "Google and Bing layers aren't available for the snapshot, due to copyright restrictions.",
-            "downloadingSnapshots": "pending snapshot generation",
-            "date": "Date",
-            "layers": "Layers",
-            "size": "Size",
-            "notsupported": "Snapshot not supported",
-            "taintedMessage": "Save snapshot functionality is limited for browser security limitations. For better result right-click on the preview and select 'Save image as' (supported by Firefox and Chrome)"
+            "title": "Snapshot Vorschau",
+            "save": "Speichern",
+            "tooltip": "Speichere einen Snapshot der Karte.",
+            "googleBingError": "Google und Bing Ebenen sind aufgrund urheberrechtlicher Einschränkungen für den Snapshot nicht verfügbar.",
+            "downloadingSnapshots": "Snapshot Erstellung...",
+            "date": "Datum",
+            "layers": "Ebenen",
+            "size": "Größe",
+            "notsupported": "Snapshot nicht unterstützt",
+            "taintedMessage": "Die Snapshot Speichern Funktionalität ist aufgrund von Browser Sicherheits Beschränkungen eingeschränkt. Für ein besseres Ergebnis klicke mit der rechten Maustaste auf die Vorschau und wähle "Bild speichern unter" (unterstützt von Firefox und Chrome)"
         },
```

Comment:
use ' instead of " inside the string and use " " to wrap it.
"Bild speichern unter" into 'Bild speichern unter'